packages/language-server/src/plugins/svelte/features/getCompletions.ts

```diff
@@ -8,9 +9,21 @@ import {
 } from 'vscode-languageserver';
 import { SvelteTag, documentation, getLatestOpeningTag } from './SvelteTags';
 import { isInTag } from '../../../lib/documents';
-/**
- * Get completions for special svelte tags within moustache tags.
- */
+
+const HTML_COMMENT_START = '<!--';
+
+const componentDocumentationCompletion: CompletionItem = {
+    label: '@component',
+    insertText: `component${EOL}$1${EOL}`,
+    documentation: 'documentation for this component. ' +
+        'It will show up on hover. You can use markdown or code block here',
```

Comment:
```suggestion
    documentation: 'Documentation for this component. ' +
        'It will show up on hover. You can use markdown and code blocks here',
```